src/java/org/apache/cassandra/security/FileBasedSslContextFactory.java

```diff
@@ -174,6 +170,30 @@ protected TrustManagerFactory buildTrustManagerFactory() throws SSLException
         }
     }
 
+    private KeyManagerFactory getKeyManagerFactory(final boolean isOutboundKeystore) throws SSLException
```

Comment:
Why not pass in the keystore context directly rather than go through the indirection of the `isOutboundKeystore` boolean? I don't see it used for anything else.